playbooks/roles/flannel-network/tasks/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
 - name: "install flannel-network"
-  command: kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/v0.9.1/Documentation/kube-flannel.yml
+  command: kubectl apply -f https://raw.githubusercontent.com/coreos/flannel/master/Documentation/kube-flannel.yml
```

Comment:
I think it would be better to use a pinned version to avoid surprises when things change?

Reply:
There is no tag available after 0.10.0, the next best thing is to use the latest commit sha for the file we need.